.github/workflows/unit-test.yml

```diff
@@ -125,32 +131,42 @@ jobs:
       - name: Report Coverage
         if: matrix.node == '14'
         run: npm run codecov:ci:changed
-  browser-tests:
+
+  browser-test:
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Why matrix when there is only one version? Shouldn't we just run setup-node with `12` directly?

Reply:
I did it that way to make sure all the setup and env vars match between the jobs. To make it less error-prone to bring changes over from one job to another.

Actually, it does not make sense to have them as separate jobs at all - the bootstrapping and install phase is more or less the same. I'll see whether we can just skip one of those steps next.

Reply:
@dyladan, your thoughts?

Reply:
if we can merge them into a single matrix that would be nice but iirc finding an image that worked well with chrome and such was a pain and using different images for different branches of a matrix is also a pain.

I'm fine with the explanation you gave. There is no real _disadvantage_ to having a single-branch matrix.